packages/client/src/workflow-client.ts

```diff
@@ -1225,6 +1227,10 @@ export class WorkflowClient extends BaseClient {
           : undefined,
       cronSchedule: options.cronSchedule,
       header: { fields: headers },
+      userMetadata: {
+        summary: jsonConverter.toPayload(options?.staticSummary),
```

Comment:
You will need to use `await encodeToPayload(...)` instead here and everything else in the client package. Not in Workflows and Worker, though.

Reply:
yup, done